README.md

```diff
@@ -1,2 +1,42 @@
-# Hello-Microverse
-first linter project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [ ] You should rename the `Project Name` placeholder text with the name of your project. e.g. `Hello Microverse` in this case